### ISSUE TITLE
How to Setup HA (CFE) in Azure where our BIG-IP F5 is already running on Single NIC in Azure

### ISSUE THREAD

**Issue by vinod1190** [F5Networks/f5-cloud-failover-extension#143]
## Do you already have an issue opened with F5 support?
Github Issues are consistently monitored by F5 staff, but should be considered as best effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open an case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

## Description
How to Setup HA (CFE) in Azure where our BIG-IP F5 is already running on Single NIC in Azure

## Environment information
For bugs, enter the following information:
- Cloud Failover Extension Version:
- BIG-IP version:
- Cloud provider:

## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: <Fill in level: 1 through 5>

Severity level definitions:
 1. **Severity 1 (Critical)** : Defect is causing systems to be offline and/or nonfunctional. immediate attention is required.
 2. **Severity 2 (High)** : Defect is causing major obstruction of system operations.
 3. **Severity 3 (Medium)** : Defect is causing intermittent errors in system operations.
 4. **Severity 4 (Low)** : Defect is causing infrequent interuptions in system operations.
 5. **Severity 5 (Trival)** : Defect is not causing any interuptions to system operations, but none-the-less is a bug.
 
 How to setup this in Azure CFE Prerequisites
 
 1:- Virtual addresses created in a floating traffic group and matching addresses (secondary) on the IP configurations of the instance NICs serving application traffic
 2:- We have single NIC in our existing F5 VMs, So can you please explain bit more step by step how to configure CFE  in existing running BIG IP F5

**Comment by mikeshimkus** [comment]
Hi @vinod1190, the documentation here should have everything you need: https://clouddocs.f5.com/products/extensions/f5-cloud-failover/latest/userguide/azure.html

If there are specific questions about configuration, let us know.

**Comment by vinod1190** [comment]
Yes I follow the same link and did the setup in our azure subscription but I can see still standalone in Device Management-->Device. Could you help me how can we check or you need any output I will share.

**Comment by mikeshimkus** [comment]
That is not really related to CFE, it is a pre-requisite. You can follow the steps here to set up a cluster: https://techdocs.f5.com/en-us/bigip-14-1-0/big-ip-device-service-clustering-administration-14-1-0/creating-an-active-standby-configuration-using-the-setup-utility.html#GUID-2B766F81-3F76-4C42-9A5C-C60BF590ADC5
